client/lib/gsuite/index.js

```diff
@@ -216,7 +226,6 @@ function isGSuiteRestricted() {
 
 export {
 	canDomainAddGSuite,
-	formatPrice,
```

Comment:
I've removed this export since that function is never used outside of this file. Less is more.